receiver/jmxreceiver/supported_jars.go

```diff
@@ -42,6 +42,10 @@ func oldFormatProperties(c *Config, j supportedJar) error {
 // If you change this variable name, please open an issue in opentelemetry-java-contrib
 // so that repository's release automation can be updated
 var jmxMetricsGathererVersions = map[string]supportedJar{
+	"2ce5255ee07e139b93011f780d814dbe9440bf1eded35f6bd5c7776dbc3fcd70": {
+		version: "1.17.0-alpha",
```

Comment:
I noticed the versions below don't have `-alpha`, so maybe the `-alpha` was being added somewhere else, in which case maybe this is breaking and will look for `1.17.0-alpha-alpha`?

Reply:
That particular version field is currently primarily maintainer documentation of the currently supported versions, and not enforced in any way. The only thing being enforced is that the hash of the provided jar matches something in the list.

Reply:
@dehaansa let's be consistent. use `-alpha` everywhere or nowhere :)

Reply:
Added this and other missing versions (and added `-alpha`) in new PR https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14696